tests/features/collections/topics.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Topics listing page
+
+  Scenario: View an index of topics
+
+    When I open `/en/collections/topics`
+    Then I see a 24 `content card`s
+    And I see a `pagination navigation`
+    And I am on an accessible page
+
+  Scenario: Open a topic
+
+    When I open `/en/collections/topics`
+    Then I see a `content card` in the `topics listing page`
+    And I click a `content card`
+    And I wait for the page to load
+    Then I should not be on the `topics listing page`
```

Comment:
I doubt this'll work yet as the 'topics listing page' isn't defined yet.
Instead use
```
  Then I see an `entity page`
```
All the entity pages should have that as a qa element.

However for that to work see the changes to the contentCard/entityCard urls first, otherwise these links open in a new window and the test fails.

Reply:
@lbiedinger I have updated the feature test but it's still failing on this scenario
`3) Scenario: Open a topic # tests/features/collections/topics.feature:9
   ✔ Before # tests/features/config/cucumber.conf.js:73
   ✖ And I click a `content card` # tests/features/support/step-definitions.js:68
       Timed out while waiting for element <[data-qa="content card"]> to be present for 5000 milliseconds. - expected "visible" but got: "not found" (5086ms)`